src/include/batch_request.h

```diff
@@ -103,6 +103,18 @@ struct rq_manage {
 	pbs_list_head rq_attr; /* svrattrlist */
 };
 
+struct rq_deljob {
+	int rq_cmd;
```

Comment:
Use rq_manage for rq_manage for deljob and remove this duplicate members from this struct, we need only rq_count and rq_jobids in this struct